binnacle/lib/ui/BinnacleHeadingUI.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+import 'package:sos/utils/Math.dart';
+import 'ArrowPainter.dart';
+
+class BinnacleHeadingUI extends StatelessWidget {
+  BinnacleHeadingUI({Key key, this.heading, this.color}) : super(key: key);
+
+  final double radius = 100;
+
+  final double heading;
+  final Color color;
+
+  Widget build(BuildContext context) {
+    return Transform.rotate(
+        angle: degreesToRadians((heading ?? 0) / 360),
+        child: new Align(
+            alignment: Alignment.topCenter,
+            child: CustomPaint(
+                painter: ArrowPainter(color: color, percentChange: 0.5))));
```

Comment:
Should all the closing parentheses be on the same line like this? This might make it harder to trace the nested UI elements.

Reply:
my IDEA is autofixing it to that, but it shows the levels on the paranthesis in order.

I'll look at this style stuff after I figure out the re-rendering issue